core/src/main/java/org/elasticsearch/cluster/metadata/IndexMetaData.java

```diff
@@ -1217,6 +1218,7 @@ public static IndexMetaData fromXContent(XContentParser parser) throws IOExcepti
      * {@link #isIndexUsingShadowReplicas(org.elasticsearch.common.settings.Settings)}.
      */
     public static boolean isOnSharedFilesystem(Settings settings) {
+        // don't use the settings directly, not to trigger manny deprecation logging
```

Comment:
Nits: `settings` -> `setting` and `not to trigger manny` -> `to not trigger verbose` (it doesn't have to be exactly this, but at least fix `manny`)